sonar-php-plugin/src/main/java/org/sonar/plugins/php/SymbolScanner.java

```diff
@@ -50,9 +51,13 @@ String name() {
   @Override
   void scanFile(InputFile file) {
     PhpFileImpl phpFile = new PhpFileImpl(file);
-    Tree ast = parser.parse(phpFile.contents());
-    SymbolTableImpl symbolTable = SymbolTableImpl.create((CompilationUnitTree) ast, new ProjectSymbolData(), phpFile);
-    symbolTable.classSymbolDatas().forEach(projectSymbolData::add);
+    try {
+      Tree ast = parser.parse(phpFile.contents());
+      SymbolTableImpl symbolTable = SymbolTableImpl.create((CompilationUnitTree) ast, new ProjectSymbolData(), phpFile);
+      symbolTable.classSymbolDatas().forEach(projectSymbolData::add);
+    } catch (RecognitionException e) {
+      LOG.debug("Parsing error in " + file);
```

Comment:
Good idea making this "debug" and simplified compared to the other parsing error logging. That way we will not have all those errors appear twice.